src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Embeddable
+public class Line extends BaseEntity{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String name;
+    @Column(unique = true)
+    private String color;
+    @Embedded
+    private Stations stations = new Stations();
+    @OneToMany(mappedBy = "line")
+    private List<Section> sections = new ArrayList<>();
+
+    protected Line() {
```

Comment:
```suggestion
    protected Line() {}
```
[컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 쓸 수도 있을 것 같습니다...!